04/app.js

```diff
@@ -0,0 +1,49 @@
+const divElement = document.querySelectorAll('div');
+console.log(divElement)
+
+
+const addClass = function () {
+
+    this.classList.add('clicked');
+
```

Comment:
w tym miejscu powinien być `setTimeout()`, zwróć uwagę, że jak klikniesz na środkowy div (najbardziej zagnieżdżony) to 3 razy wykona się ta funkcja (dla każdego div-a osobno).